FirebaseRemoteConfig/Tests/Unit/RCNRemoteConfigTest.m

```diff
@@ -1834,6 +1834,32 @@ - (void)testFetchAndActivateRolloutsNotifyInterop {
   [self waitForExpectations:@[ notificationExpectation ] timeout:_expectationTimeout];
 }
 
+- (void)testSetCustomSingals {
```

Comment:
Can we have another test case where within one rc instance call `setCustomSignals` multiple times?

Reply:
I have added multiple tests, including those that call setCustomSignals multiple times with String, Integer, and Null values. I have also added tests to check for invalid inputs.